### PR TITLE
Usr/jasteph/debugdoc

### PR DESCRIPTION
**Pull request by jasonstephen15**
Adding in VSCode Section to debugging doc

### CODE REVIEW

**Comment by jasonstephen15** [comment]
@Reezaali I know we already made some revisions in a previous word doc, was wondering if you still had that handy?

**Review by DavidShoe — APPROVED** [review]

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/howto/debug.md [hunk 1]
```diff
@@ -92,6 +92,164 @@ Complete the following actions to debug your WebView2 application.
...
+        
+1.  Open the **Debug Console**, and view any errors from the debugger.  
+    
+    :::image type="complex" source="./media/resultsvs.png" alt-text=" Visual Studio Code Debug Console" lightbox="./media/resultsvs.png":::
+       Visual Studio Code Debug Console  
+    :::image-end:::  
+    
+**Advanced Settings**:  
+
+*   Targeted Webview debugging.
```
[the remark this answers is not in the document]
Comment (Reezaali) [reply microsoft-edge/webview2/howto/debug.md:151]: In some WebView2 applications, you may use more than one WebView2 control. To pick the WebView2 control to debug in this situation you can use targeted webview2 debugging

**Review by Reezaali — APPROVED** [review]

**Comment by jasonstephen15** [comment]
#sign-off

**Comment by jasonstephen15** [comment]
#sign-off

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**